.travis.yml

```diff
@@ -3,9 +3,8 @@ php:
   - 7.2
   - 7.3
   - 7.4
-script:
-  - vendor/bin/tester tests -s -p php
-
 before_script:
-  # Install Nette Tester
   - composer install --no-interaction --dev --prefer-source
+script:
+  - vendor/bin/tester tests -s -p php
+  - vendor/bin/phpcs --standard=PSR12 --warning-severity=0 ./src
```

Comment:
I think it would be great to check coding style for `tests` folder.

IMHO, I think it will be great to check coding style before doing tester.